README.md

```diff
@@ -14,3 +14,9 @@ Development Installation
 1. Make sure you have the development dependencies installed
 2. Place `septa_bus.zip`, `septa_rail.zip`, and `patco.zip` in the gtfs folder
 3. Run `vagrant up`
+4. `vagrant ssh`
+5. `. /app_env/activate`
```

Comment:
No, `. /app_env/activate` is correct. /app_env/activate is not executable and `. ` tells bash to source the file.